selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/organization/OrganizationListPage.java

```diff
@@ -175,7 +176,7 @@ public String getOrganizationsToolbarTitle() {
   public boolean isAddOrganizationButtonVisible() {
     List<WebElement> elements =
         seleniumWebDriver.findElements(By.xpath(Locators.ADD_ORGANIZATION_BUTTON_XPATH));
-    return elements.isEmpty() ? false : elements.get(0).isDisplayed();
+    return !elements.isEmpty() && elements.get(0).isDisplayed();
```

Comment:
It's better to add check of elements size `elements.size() > 0` before getting first element to avoid potential **NullPointerException**.